tests/test_public_service.py

```diff
@@ -1,7 +1,210 @@
+                    dct:identifier     "fdk-1" ;
+                    dct:issued         "2020-10-05T13:15:39.831Z"^^xsd:dateTime ;
+                    dct:modified       "2020-10-05T13:15:39.831Z"^^xsd:dateTime ;
+                    foaf:primaryTopic  <http://public-service-publisher.fellesdatakatalog.digdir.no/services/1> ."""
+
+    expected = {
+        "http://public-service-publisher.fellesdatakatalog.digdir.no/services/1": PublicService(
+            id="fdk-1",
+            uri="http://public-service-publisher.fellesdatakatalog.digdir.no/services/1",
+            identifier="1",
+            title={"nb": "Ei offentleg teneste"},
+            description={
+                "nn": "Ei offentleg teneste som tener som døme til bruk i utvikling"
+            },
+            isGroupedBy=[
+                Event(
+                    uri="http://public-service-publisher.fellesdatakatalog.digdir.no/events/1",
+                    identifier="1",
+                    title={"nb": "Starte og drive bedrift"},
+                    description={
+                        "nb": "Elektronisk prosess for etablering og oppstart av en bedrift."
+                    },
+                    type="https://data.norge.no/concepts/1",
+                )
+            ],
+            hasCompetentAuthority=[
+                Publisher(
+                    uri="https://organizations.fellesdatakatalog.digdir.no/organizations/123456789",
+                    id="123456789",
+                    name="Digitaliseringsdirektoratet",
+                    orgPath="/STAT/987654321/123456789",
+                    prefLabel={
+                        "en": "Norwegian Digitalisation Agency",
+                        "nn": "Digitaliseringsdirektoratet",
+                        "nb": "Digitaliseringsdirektoratet",
+                    },
+                    organisasjonsform="ORGL",
+                )
+            ],
+            harvest=HarvestMetaData(
+                firstHarvested="2020-10-05T13:15:39Z", changed=["2020-10-05T13:15:39Z"]
+            ),
+            keyword=[{"nb": "Serveringsbevilling"}],
+            sector=[
+                SkosConcept(
+                    uri="https://data.norge.no/concepts/2",
+                    prefLabel={"nb": "Nacekode: 56.1"},
+                    extraType=None,
+                )
+            ],
+            isClassifiedBy=[
+                SkosConcept(
+                    uri="https://data.norge.no/concepts/16",
+                    prefLabel={"nb": "Kafé"},
+                    extraType=None,
+                ),
+                SkosConcept(
+                    uri="https://data.norge.no/concepts/17",
+                    prefLabel={"nb": "Spiserestaurant"},
+                    extraType=None,
+                ),
+            ],
+            language=[
+                SkosCode(
+                    uri="http://publications.europa.eu/resource/authority/language/NOB"
```

Comment:
Tror du trenger å legge til at language oppdateres fra reference_data

Reply:
Har lagt til extend-funksjon for language.